### PR TITLE
[nojira] Small tweaks to travis config

### PR DESCRIPTION
**Pull request by matthewdavidson**
see comments inline

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +22,7 @@ script:
 cache:
   directories:
     - node_modules
+    - native/node_modules
```
Comment (matthewdavidson) [inline .travis.yml]: Something i noticed, there is no reason why we cant cache the native `node_modules` too

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
```
Comment (matthewdavidson) [inline .travis.yml]: travis automatically picks up the version from `.nvmrc` - no need to over specify nd give ourselves more work. We do this in other repos too btw.

File: .travis.yml (hunk 1 above)
Comment (shaundon) [reply .travis.yml]: Yep good idea!

File: .travis.yml (hunk 2 above)
Comment (matthewdavidson) [reply .travis.yml]: See https://travis-ci.org/Skyscanner/backpack/builds/427659336#L432

**Pull request merged**